include/s3select_oper.h

```diff
@@ -15,6 +15,14 @@
 #include <boost/bind.hpp>
 #include "s3select_parquet_intrf.h" //NOTE: should include first (c++11 std::string_view)
 
+#if REGEX_HS
```

Comment:
s3select CMake is not used by RGW CMake files.
the RGW is using only the s3select header files.

the header file can validate whether RE2/HS exists, by using the `__has_include (<....>)`  operator.
[https://github.com/ceph/s3select/blob/1609bb2ab5441d2314f56858f4f98fc2be509f89/include/s3select_parquet_intrf.h#L4](url)
 


the `REGEX_EN` definition can be used internally by s3select CMAKE, for selecting regex engine (for testing)

Reply:
Done